lib/src/services/yust_database_service_flutter.dart

```diff
@@ -302,11 +303,9 @@ class YustDatabaseService {
     List<YustFilter>? filters,
     int? limit,
   }) async {
-    throw YustException('Not implemented for flutter');
-    // Wait for https://github.com/firebase/flutterfire/pull/11757 to be merged
-    // var query = getQuery(docSetup, filters: filters);
-    // final snapshot = await query.average(fieldPath).get();
-    // return snapshot.count;
+    var query = getQuery(docSetup, filters: filters);
+    final snapshot = await query.aggregate(cf.average(fieldPath)).get();
```

Comment:
I don't think you need `cf` here.
Also can you test if this works with fieldPaths containing numeric parts, e.g. `test.01234test`